ai-ml/trainium-inferentia/eks.tf

```diff
@@ -176,9 +176,9 @@ module "eks" {
         echo "Bootstrap complete. Ready to Go!"
       EOT
 
-      min_size     = 0
-      max_size     = 2
-      desired_size = 0
+      min_size     = 4
+      max_size     = 4
+      desired_size = 4
```

Comment:
We are using the same node group for multiple LLMs and the min_size of node groups are different to each one. 
Could you change this back to 
```
min_size     = 0
desired_size = 0
```
so that we can ask the users to update the min and desired before running `install.sh`.  Add this in install instructions of the website or blog.

Reply:
I did change this to variables var.min_size, var.max_size, var.desired_size. Set default to zero. Did not modify the code for core node group

Reply:
Who is running this can do something like this: export TF_VAR_min_size=x

Reply:
> We are using the same node group for multiple LLMs and the min_size of node groups are different to each one. Could you change this back to
> 
> ```
> min_size     = 0
> desired_size = 0
> ```
> 
> so that we can ask the users to update the min and desired before running `install.sh`. Add this in install instructions of the website or blog.

Yes I changed this back to 0